app/views/Project/Tagging/Sources/SourcesTable/index.tsx

```diff
@@ -70,11 +69,94 @@ const defaultSorting = {
 
 interface Props {
     className?: string;
-    projectId: number;
-    filters?: Filters;
+    projectId: string;
+    filters: Omit<ProjectSourcesQueryVariables, 'projectId'>;
     refreshTimestamp: number | undefined;
 }
 
+export const PROJECT_ENTRIES = gql`
+    query ProjectSources(
+        $projectId: ID!,
+        $page: Int,
+        $pageSize: Int,
+        $ordering: String,
+        $assignees: [ID!],
+        $authoringOrganizationTypes: [ID!],
+        $confidentiality: LeadConfidentialityEnum,
+        $createdAt_Gte: DateTime,
+        $createdAt_Lt: DateTime,
+        $emmEntities: String,
+        $emmKeywords: String,
+        $emmRiskFactors: String,
+        $exists: LeadExistsEnum,
+        $priorities: [LeadPriorityEnum!],
+        $publishedOn_Gte: Date,
+        $publishedOn_Lt: Date,
+        $search: String,
+        $statuses: [LeadStatusEnum!],
+        $entriesFilterData: LeadEntriesFilterData,
+        ) {
+        project(id: $projectId) {
+            leads (
+                page: $page,
+                pageSize: $pageSize,
+                ordering: $ordering,
+                assignees: $assignees,
+                authoringOrganizationTypes: $authoringOrganizationTypes,
+                confidentiality: $confidentiality,
+                createdAt_Gte: $createdAt_Gte,
+                createdAt_Lt: $createdAt_Lt,
+                emmEntities: $emmEntities,
+                emmKeywords: $emmKeywords,
+                emmRiskFactors: $emmRiskFactors,
+                exists: $exists,
+                priorities: $priorities,
+                publishedOn_Gte: $publishedOn_Gte,
+                publishedOn_Lt: $publishedOn_Lt,
+                search: $search,
+                statuses: $statuses,
+                entriesFilterData: $entriesFilterData,
+            ) {
+                totalCount
+                page
+                pageSize
+                results {
+                    id
+
+                    confidentiality
+                    clientId
+                    status
+                    createdAt
+                    title
+                    publishedOn
+                    priority
+                    createdBy {
+                        displayName
+                    }
+                    project {
+                        id
+                    }
+                    authors {
+                        title
+                    }
+                    assignee {
+                        displayName
+                    }
```

Comment:
```suggestion
                    createdBy {
                        id
                        displayName
                    }
                    project {
                        id
                    }
                    authors {
                        id
                        title
                    }
                    assignee {
                        id
                        displayName
                    }
```

Always get `id` field so that apollo client can properly update the normalized cache